packages/rn-tester-e2e/tests/screens/components/buttonComponent.screen.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import {UtilsSingleton as Utils} from '../../helpers/utils';
+
+class ButtonComponentScreen {
+  // reference in the Components list
+  buttonScreenElement: string = Utils.platformSelect({
+    ios: '[label="Button"]',
```

Comment:
I think we can have better API's here... 
like
```
// in Utils

function iOSLabel(label: string): string {
  return `[label="${label}"]`;
}

// Used like this.
Utils.iOSLabel("Button")
```

Reply:
Hi @kelset,
This is fixed in the last MR. Could you please resolve this comment?